e2e/e2e-create-windows-nodepool.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+source e2e-helper.sh
+
+log "Starting to create windows nodepool"
+
+# windowsNodepool="win19"
```

Comment:
Could we remove these?

Reply:
Miss the redundant code and will remove them all.